src/main/kotlin/io/csbroker/apiserver/common/util/CookieUtil.kt

```diff
@@ -32,13 +36,30 @@ fun deleteCookie(request: HttpServletRequest, response: HttpServletResponse, nam
 
 fun OAuth2AuthorizationRequest.serialize(): String {
     return Base64.getUrlEncoder()
-        .encodeToString(SerializationUtils.serialize(this))
+        .encodeToString(this.toByteArray())
 }
 
-fun <T> deserialize(cookie: Cookie, cls: Class<T>): T {
-    return cls.cast(
-        SerializationUtils.deserialize(
-            Base64.getUrlDecoder().decode(cookie.value),
-        ),
-    )
+fun <T> deserialize(cookie: Cookie): T {
+    return Base64.getUrlDecoder().decode(cookie.value).fromByteArray()
+}
+
+@Suppress("UNCHECKED_CAST")
+private fun <T : Serializable> ByteArray.fromByteArray(): T {
+    val byteArrayInputStream = ByteArrayInputStream(this)
+    val objectInput = ObjectInputStream(byteArrayInputStream)
+    val result = objectInput.readObject() as T
+    objectInput.close()
+    byteArrayInputStream.close()
+    return result
+}
+
+private fun Serializable.toByteArray(): ByteArray {
+    val byteArrayOutputStream = ByteArrayOutputStream()
+    val objectOutputStream = ObjectOutputStream(byteArrayOutputStream)
+    objectOutputStream.writeObject(this)
+    objectOutputStream.flush()
+    val result = byteArrayOutputStream.toByteArray()
+    byteArrayOutputStream.close()
+    objectOutputStream.close()
+    return result
```

Comment:
```suggestion
    return ByteArrayOutputStream().use { byteArrayOutputStream  ->
        ObjectOutputStream(byteArrayOutputStream).use { objectOutputStream ->
            objectOutputStream.writeObject(this)
            objectOutputStream.flush()
            byteArrayOutputStream.toByteArray()
        }
    }
```
코틀린에서 자원관리를 대신 해주는 use 메서드를 사용해보는 게 어떨까요~!